manifests/w/WHONET/WHONET2023/23.9.13/WHONET.WHONET2023.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WHONET.WHONET2023

```

```suggestion
PackageIdentifier: WHONET.WHONET.2023
```
When there is a year as part of the identifier, it should be made a sub-package by adding a `.`
After changing this value, the manifest will need to be moved to the appropriate folder